keyboards/handwired/slash/keymaps/default/layers.json

```diff
@@ -0,0 +1,120 @@
+[
```

Comment:
What's the purpose of this file?

Reply:
Not sure where/what its from, I removed it in the latest commit